server/src/main/java/org/candlepin/pinsetter/tasks/HypervisorUpdateJob.java

```diff
@@ -393,14 +371,15 @@ public static String decompress(byte[] bytes) {
     /*
      * Create a new hypervisor type consumer to represent the incoming hypervisorId
      */
-    private Consumer createConsumerForHypervisorId(String incHypervisorId, Owner owner, Principal principal) {
+    private Consumer createConsumerForHypervisorId(String incHypervisorId, Owner owner, Principal principal,
+                                                   Consumer incoming) {
         Consumer consumer = new Consumer();
         consumer.setName(sanitizeHypervisorId(incHypervisorId));
         consumer.setType(this.hypervisorType);
         consumer.setFact("uname.machine", "x86_64");
         consumer.setGuestIds(new ArrayList<GuestId>());
         consumer.setOwner(owner);
-        consumer.setAutoheal(true);
+        consumer.setAutoheal(!owner.getAutobindDisabled());
```

Comment:
I don't think this is what this setting is for. Isn't the owner.AutobindDisabled used for maintenance mode?

Reply:
My intention was to set the autobind to the default for the org. I can stick with the default of "true" for now if so desired.

Reply:
Changed back to default of "true".